cmd/dep/status.go

```diff
@@ -214,11 +214,24 @@ func (cmd *statusCommand) Run(ctx *dep.Ctx, args []string) error {
 		}
 	}
 
-	if err := runStatusAll(ctx.Loggers, out, p, sm); err != nil {
+	memoMismatch, hasMissingPkgs, err := runStatusAll(ctx.Loggers, out, p, sm)
+	if err != nil {
 		return err
 	}
 
-	ctx.Loggers.Out.Print(buf.String())
+	if memoMismatch {
+		if hasMissingPkgs {
+			ctx.Loggers.Err.Println("Lock memo mismatch due to the following missing packages:\n")
```

Comment:
Let's clarify where the packages are missing, like `Lock memo mismatch due to the following packages missing from the lock:`